src/coins.h

```diff
@@ -197,11 +201,11 @@ struct CCoinsCacheEntry
     }
 
     //! Only use this for initializing the linked list sentinel
-    inline void SelfRef(CoinsCachePair& self) noexcept
+    inline void SelfRef(CoinsCachePair& pair) noexcept
```

Comment:
minor nit: I don't think this one needed to be renamed to `pair`. `self` still makes sense here.

Reply:
yeah, but it's more consistent this way (and method name already mentions self)